cms/static/js/spec/views/course_video_settings_spec.js

```diff
@@ -438,30 +508,27 @@ define(
                 verifyOrganizationCredentialsView();
             });
 
-            it('shows api secret input field if selected provider is 3Play Media', function() {
-                // Set selected provider to 3Play Media
-                changeProvider('3PlayMedia');
-
+            it('shows cielo specific organization credentials fields only', function() {
+                verifyProviderSelectedView();
                 // Click change button to render organization credentials view.
                 $courseVideoSettingsEl.find('.action-change-provider').click();
 
-                // Verify 3play api secret and api key are present.
-                verifyOrganizationCredentialField('api-secret', 'API Secret');
-                verifyOrganizationCredentialField('api-key', 'API Key');
+                // Verify api key is present.
+                verifyCredentialFieldsPresent({
+                    'api-key': 'API Key',
+                    username: 'Username'
+                });
             });
 
-            it('does not show api secret input field if selected provider is not 3Play Media', function() {
-                verifyProviderSelectedView();
-                // Click change button to render organization credentials view.
-                $courseVideoSettingsEl.find('.action-change-provider').click();
-
-                // Verify 3Play Media api secret is not present.
-                expect(
-                    $courseVideoSettingsEl.find('.' + courseVideoSettingsView.selectedProvider + '-api-secret')
-                ).not.toExist();
+            it('shows 3play specific organization credentials fields only', function() {
+                // Set selected provider to 3Play Media
+                changeProvider('3PlayMedia');
```

Comment:
@mushtaqak Do we also need to call `verifyProviderSelectedView` here?

Reply:
No `selectedView` would not be shown here, because here, we have just changed the provide (Clicked on "change" and selected "3play"), so that it would reveal credential input fields for "3play".